themes/default/content/docs/reference/service-api/_index.md

```diff
@@ -0,0 +1,1465 @@
+        "type": "pulumi:providers:aws",
+        "inputs": {
+          "region": "us-west-2",
+          "version": "3.32.0"
+        },
+        "outputs": {
+          "region": "us-west-2",
+          "version": "3.32.0"
+        }
+      },
+      {
+        "urn": "urn:pulumi:dev-user1::demo-aws-ts-webserver::aws:ec2/vpc:Vpc::main",
+        "custom": true,
+        "id": "vpc-04057e18647a66e24",
+        "type": "aws:ec2/vpc:Vpc",
+        "inputs": {
+          "__defaults": [
+            "assignGeneratedIpv6CidrBlock",
+            "enableDnsSupport",
+            "instanceTenancy"
+          ],
+          "assignGeneratedIpv6CidrBlock": false,
+          "cidrBlock": "10.0.0.0/16",
+          "enableDnsSupport": true,
+          "instanceTenancy": "default"
+        },
+        "outputs": {
+          "__meta": "{\"schema_version\":\"1\"}",
+          "arn": "arn:aws:ec2:us-west-2:0123456789:vpc/vpc-04057e18647a66e24",
+          "assignGeneratedIpv6CidrBlock": false,
+          "cidrBlock": "10.0.0.0/16",
+          "defaultNetworkAclId": "acl-0b63012743b57140d",
+          "defaultRouteTableId": "rtb-01c36ce5fcc3ac797",
+          "defaultSecurityGroupId": "sg-072f0a9bf075de1f5",
+          "dhcpOptionsId": "dopt-6db97615",
+          "enableClassiclink": false,
+          "enableClassiclinkDnsSupport": false,
+          "enableDnsHostnames": false,
+          "enableDnsSupport": true,
+          "id": "vpc-04057e18647a66e24",
+          "instanceTenancy": "default",
+          "ipv6AssociationId": "",
+          "ipv6CidrBlock": "",
+          "mainRouteTableId": "rtb-01c36ce5fcc3ac797",
+          "ownerId": "0123456789",
+          "tags": {}
+        },
+        "parent": "urn:pulumi:dev-user1::demo-aws-ts-webserver::pulumi:pulumi:Stack::demo-aws-ts-webserver-dev-user1",
+        "provider": "urn:pulumi:dev-user1::demo-aws-ts-webserver::pulumi:providers:aws::default_3_32_0::d77c9fd7-1c06-427d-ba26-dbd8a8cbf91a",
+        "propertyDependencies": {
+          "cidrBlock": null
+        }
+      }
+    ]
+  }
+}
+```
+<!-- ###################################################################### -->
+
+## Stack Tags
+
+### Get Stack Tags
+
+Use [Get Stack](#get-stack) to retrieve a stack's details including its tags.
+
+### Set Stack Tag
+
+```
+POST https://api.pulumi.com/api/stacks/{organization}/{project}/{stack}/tags
+```
+
+#### Parameters
+
+| Parameter | Type | In | Description |
+| --------- | ---------- | ---------- | ---------- |
+| `organization` | string | path | organization name |
+| `team` | string | path | team name |
+| `stack` | string | path | stack name |
+| `name` | string | body | tag name |
+| `value` | string | body | tag value |
+
+#### Example
+
+```bash
+curl \
+  -H 'Accept: application/vnd.pulumi+3' \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: token pul-abcdefghijklmnopqrstuvwxyz' \
+  --request POST \
+  --data '{"name":"{tagName}","value":"{tagValue}"}' \
```

Comment:
Having the "Parameters" table makes this a little bit confusing.

As there are URL _route parameters_ (e.g. parts of the REST-ful URL). And there is the _POST request body_, which contains the JSON blob with a `name` and `value` field. Perhaps we'd want to call that out somehow, rather than putting it all into the same "Parameters" table? Also, in some places we accept URL-query parameters as well, which is a third class of "ways you pass data when making a REST API call".

Reply:
Going to leave this as-is per our convo.